frontend/src/components/smart/SkillBalanceDisplay.vue

```diff
@@ -241,13 +241,13 @@ export default Vue.extend({
     return{
       ClaimStage,
       skillAmount: 0,
-      goldAmount: 0,
+      ValorAmount: 0,
```

Comment:
In many places, case error, goldX was changed to ValorX. Please check and correct to omit unnecessary bugs